src/test/regress/expected/multi_view.out

```diff
@@ -913,7 +905,7 @@ EXPLAIN (COSTS FALSE) SELECT et.* FROM recent_10_users JOIN events_table et USIN
  Limit
    ->  Sort
          Sort Key: remote_scan."time" DESC
-         ->  Custom Scan (Citus Real-Time)
+         ->  Custom Scan (Citus Adaptive)
```

Comment:
Do you know why were these Real-Time previously?

Reply:
I generated the file with check-multi, it's been left with Real-Time since adaptive executor was brought in due to normalization